Detectors/ITSMFT/ITS/macros/test/CheckTracks.C

```diff
@@ -305,8 +305,12 @@ void CheckTracks(std::string tracfile = "o2trac_its.root", std::string clusfile
       }
 
       Int_t mID = mcTrack.getMotherTrackId();
-      if (mID >= 0)
-        continue; // Select primary particles
+      if (mID >= 0) {
+        const auto& mom = (*mcArr)[mID];
+        int pdg = std::abs(mom.GetPdgCode());
+        if (pdg > 100 || (pdg < 20 && pdg > 10))
```

Comment:
but what about the primary electrons and muons? 
Right now we don't really select the primaries.

Reply:
Here just the daughters of electrons and muons are rejected, electrons and muons themself are not touched by this selection. Am I missing something?